packages/react-composites/src/composites/ChatComposite/file-sharing/FileUpload.ts

```diff
@@ -125,20 +125,20 @@ export class FileUpload implements FileUploadManager, ObservableFileUpload {
   }
 
   notifyUploadProgressed(value: number): void {
-    this._emitter.emit('uploadProgressed', this.id, value);
+    this._emitter.emit('uploadProgress', this.id, value);
   }
 
   notifyUploadCompleted(metadata: FileMetadata): void {
-    this._emitter.emit('uploadCompleted', this.id, metadata);
+    this._emitter.emit('uploadComplete', this.id, metadata);
   }
 
   notifyUploadFailed(message: string): void {
-    this._emitter.emit('uploadFailed', this.id, message);
+    this._emitter.emit('uploadFail', this.id, message);
   }
 
-  on(event: 'uploadProgressed', listener: UploadProgressListener): void;
-  on(event: 'uploadCompleted', listener: UploadCompleteListener): void;
-  on(event: 'uploadFailed', listener: UploadFailedListener): void;
+  on(event: 'uploadProgress', listener: UploadProgressListener): void;
```

Comment:
(non-blocking, just opinion) "on upload progress" sounds weird as progress to me reads as a noun here instead of a verb/action. Suggestion `uploadProgressChange` ?

Reply:
Good suggestion. Will change it